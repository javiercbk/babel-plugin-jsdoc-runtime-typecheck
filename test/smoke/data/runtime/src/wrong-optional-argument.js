// @typecheck

/**
 * @param {Number} a
 * @param {Number} [b]
 * @returns {Number}
 */
function test(a, b) {
    return a + b;
}

test(1, '2');
