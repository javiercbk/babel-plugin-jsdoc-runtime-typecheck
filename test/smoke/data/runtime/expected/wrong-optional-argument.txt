'TypeError: \n    Parameter "b" in function "test" has wrong type.\n    Expected: Number (optional)\n    Current:  "2"\n'

